lib/core/src/persist/migrations.rs

```diff
@@ -35,5 +35,24 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
             amount_sat INTEGER NOT NULL,
             fees_sat INTEGER NOT NULL
         ) STRICT;",
+        "CREATE TABLE IF NOT EXISTS chain_swaps (
+            id TEXT NOT NULL PRIMARY KEY,
+            payment_type INTEGER NOT NULL,
```

Comment:
The term payment_type belongs to a higher level of abstraction IMO. It may also have more variants in the future that may not fit here.
Perhaps we can just use a different enum? Direction{ Outgoing, Incoming} or  Direction(Send, Receive} ?